### PR TITLE
Update Trello adapter to use REST APIs for fetching ticket info

### PR DESCRIPTION
**Pull request by nickgnd**
Based on #137 

Here the documentation of the REST APIs: https://developers.trello.com/v1.0/reference#introduction

I tested the changes both with Firefox

**TODO:**
- [x] test with Chrome
- [x] test with Safari
- [x] test with Opera

### CODE REVIEW

File: src/common/adapters/trello.js [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+  if (host !== 'trello.com') return null;
+
+  const { key } = match('/c/:key/:slug', pathname);
+
+  if (!key) return null;
+
+  const trello = client('https://api.trello.com/1');
+  const response = await trello.get(`cards/${key}`).json();
+
+  const { idShort: id, name: title } = response;
```
[the remark this answers is not in the document]
Comment (pmeinhardt) [reply src/common/adapters/trello.js]: The `shortLink` however looks like a good match:

> Numeric ID for the card on this board. Only unique to the board, and subject to change as the card moves

The short link is also easy to use to open your ticket.

We just need to append a query parameter to the API request above:

```javascript
const options = { searchParams: { fields: 'name,desc,shortLink,shortUrl' } };
const response = await trello.get(`cards/${key}`, options).json();

const  { name: title, shortLink: id, shortUrl: url } = response;
```

**Comment by pmeinhardt** [comment]
I've updated the adapter (see 3c2ae1e).

Here's a preview of the generated command with branch name and commit message:

```shell
git checkout -b 'feature/wt8kdpvi-video-streaming-recording-setup' && git commit --allow-empty -m '[#wt8KDpVi] Video streaming/recording setup

For lunch talks, conferences, or meetups it would be great to have some (reliable) setup to record talks and audio. It would be great to be able to also live-stream it.

https://trello.com/c/wt8KDpVi'
```

The card `shortLink` isn't a perfect fit either (at least with the default config):
It's case sensitive, but the default branch name mapping downcases the ticket id 🤷‍♂️ 

I don't really think that's an issue though. I'd say this is good to go if you're okay with my tweaks @nickgnd. Nice work mate! ✋

File: src/common/adapters/trello.js [hunk 2]
```diff
@@ -12,20 +12,35 @@ import match from 'micro-match';
...
+    type: 'feature',
+    id,
+    title,
+    description,
+    url,
+  };
+}
 
-  if (host !== 'trello.com') return null;
+const requestOptions = { searchParams: { fields: 'name,desc,shortLink,shortUrl' } };
```
Comment (nickgnd) [inline src/common/adapters/trello.js]: 👍

**Comment by nickgnd** [comment]
@pmeinhardt Looks great! Thanks for all the work 🎉  (...and sorry for the late reply 😞 )

**Pull request merged**